sds_data_manager/stacks/instrument_lambdas.py

```diff
@@ -14,7 +14,7 @@
 from sds_data_manager.stacks.database_stack import SdpDatabase
 
 
-class InstrumentLambda(Construct):
+class InstrumentLambda(Stack):
```

Comment:
May be we should name it `BatchStarterLambda` since we have been creating other lambda on its own. and carry that name through out this lambda. Right now, I think this one lambda will build dependency for all instrument based on current input. If we decide to break down for each instrument, we can then revisit too.

I am happy to jump on call real quick to discuss what I mean.

Reply:
Nice catch!